src/ActivitiesAdapter.test.js

```diff
@@ -0,0 +1,29 @@
+import {isObservable} from 'rxjs';
+
+import ActivitiesAdapter from './ActivitiesAdapter';
+
+describe('Activities Adapter Interface', () => {
+  let activitiesAdapter;
+
+  beforeEach(() => {
+    activitiesAdapter = new ActivitiesAdapter();
+  });
+
+  test('getActivity() returns an observable', () => {
+    expect(isObservable(activitiesAdapter.getActivity())).toBeTruthy();
+  });
+
+  test('getActivity() throws a proper error message', (done) => {
```

Comment:
```suggestion
  test('getActivity() errors because it needs to be defined', (done) => {
```

To be consistent with other tests